---
title: "CRISPR-Cas9"
author: "yincy"
date: "2/8/2021"
output: html_document

bibliography: 
- bib/CRISPR.bib
- ../../Data/mendeley-bib/CRISPR.bib
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```

# What is CRISPR-Cas9?
**CRISPR** (**C**lustered **R**egularly **I**nterspaced **S**hort **P**alindromic **R**epeats) sequences and **Cas** (**C**RISPR-**as**sociated) proteins are the two elements of an ancient defence prokaryotic adaptive restriction system conserved in bacterial genomes. While CRISPRs represent the memory of the system, a repository of short, directly repeating nucleotide sequences flanked by short unique DNA fragments, acquired from previous infections, Cas proteins are the actual effectors, that are able to process the CRISPR sequences into small RNAs, and to cleave those infectious DNA molecules that perfectly match those CRISPR-derived RNAs. To translate a complex prokaryotic system into a simple genome editing tool, the crRNA (CRISPR RNA) and the tracrRNA (trans-activating crRNA) have been fused in a synthetic small guide RNA (sgRNA), composed by a hairpin RNA structure, resembling the tracrRNA, linked to a 20 bp sequence homologous to the target DNA. Out of all the CRISPR-associated proteins, <a style='color:blue'>Cas9 is the final effector, capable of complexing and cleaving both strands of a DNA molecule upon detecting a typical Watson&Crick homologous base pair match with the sgRNA. Therefore the CRISPR-Cas system is often referred as CRISPR/Cas9</a>. Genome engineering by these RNA-programmable Cas9 nucleases have broad applications in biology, biomedicine and biotechnology. The microbiologist Emmanuelle Charpentier and the structural biologist Jennifer Doudna, along with the rest of their colleagues co-authoring the first seminal publications on this subject, are the ones to be credited for having investigated and brought this amazing prokaryotic tool to the attention and for the benefit of the eukaryotic world.  

```{r, fig.align='center', fig.cap='Scheme illustrating the CRISPR-Cas9 mechanism of action'}
knitr::include_graphics("figures/CRISPR-Cas9-mechanism-of-action.PNG")
```


```{r, fig.align='center', fig.cap='Scheme illustrating the two repair pathways occuring after CRISPR-Cas9 action'}
knitr::include_graphics("figures/two-repair-pathway-of-CRISPR-Cas9.PNG")
```

# The history of CRISPR  
1. Nucleotide sequence of the iap gene, responsible for alkaline phosphatase isozyme conversion in Escherichia coli, and identification of the gene product.  
**First report in Gram^-^ bacteria**  

2. Insertion element IS987 from Mycobacterium bovis BCG is located in a hot-spot integration region for insertion elements in Mycobacterium tuberculosis complex strains.  
**First report in Gram^+^ bacteria**  

3. Transcription at different salinities of Haloferax mediterranei sequences adjacent to partially modified PstI sites.  
**First report in archeas**

4. Nature of DNA polymorphism in the direct repeat cluster of Mycobacterium tuberculosis; application for strain differentiation by a novel typing method.   
**First application: typing**

5. Long stretches of short tandem repeats are present in the largest replicons of the Archaea Haloferax mediterranei and Haloferax volcanii and could be involved in replicon partitioning.  
**First functional study**

6. Biological significance of a family of regularly spaced repeats in the genomes of Archaea, Bacteria and mitochondria.  
**First comilation of SRSR repeats in available genomes from bacteria, archea and mitochondria and first indication of their biological significance**

7. Identification of genes that are associated with DNA repeats in prokaryotes.  
**First paper where the word CRISPR is used, as proposed by Francis Mojica, first paper describing the Cas genes**

8. Identification of 86 candidates for small non-messenger RNAs from the archaeon Archaeoglobus fulgidus.  
**First paper describing the CRISPR RNAs (crRNA)**

9. Intervening sequences of regularly spaced prokaryotic repeats derive from foreign genetic elements.  
**First paper suggesting that CRISPR-Cas system are involved in bacterial immunity against foreign genetic elements**

10. Clustered regularly interspaced short palindrome repeats (CRISPRs) have spacers of extrachromosomal origin.  
**First paper finding DNA sequence conservation adjacent to protospacers, which later on would be known as PAMs**

11. CRISPR provides acquired resistance against viruses in prokaryotes.  
**First paper demonstrating that CRIPSR-Cas system provide adaptive (acquired) bacterial immunity againts bacteriophages**

12. CRISPR interference limits horizontal gene transfer in staphylococci by targeting DNA.  
**First paper demonstrating that CRISPR-Cas systems interfere with horizontal transfer of plasmids**

13. Small CRISPR RNAs guide antiviral defense in prokaryotes.  
**First paper demonstrating the guidance of crRNA to drive antiviral defense**

14. Short motif sequences determine the targets of the prokaryotic CRISPR defence system.  
**First description of Protospacer Adjacent Motifs (PAMs)**

15. Structural basis for DNase activity of a conserved protein implicated in CRISPR-mediated genome defense.  
**First paper describing the protein structure involved in DNase activity in the CRISPR-Cas system**

16. Sequence- and structure-specific RNA processing by a CRISPR endonuclease.  
**First paper describing the protein structure involved in RNA processing in the CRISPR-Cas system**

17. Sequence- and structure-specific RNA processing by a CRISPR endonuclease.  
**Fist paper describing the protein structure involved in RNA processing in the CRISPR-Cas system**

18. The CRISPR/Cas bacterial immune system cleaves bacteriophage and plasmid DNA.  
**First describing of the interference/defense mechanism: DNA target cleavage**

19. CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III.  
**First paper describing the tracrRNAs as a crytical compund of the CRISPR-Cas system**

20. Evolution and classification of the CRISPR-Cas systems.  
**First attempt to classify all known CRISPR-Cas systems**

21. The Streptococcus thermophilus CRISPR/Cas system provides immunity in Escherichia coli.  
**First transmission of a CRISPR-Cas system between two bacterial species very distantly related in evolution**

22. A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.  
**First paper using sgRNA as crRNA+tracrRNA and suggesting that CRISPR-Cas9 system could be used for "RNA-programmable genome editing"**

23. Cas9-crRNA ribonucleoprotein complex mediates specific DNA cleavage for adaptive immunity in bacteria.  
**Almost first paper suggesting CRISPR-Cas9 system to be used for "engineering of universal programmable RNA-guides DNA endonucleases"**

24. A bacteriophage encodes its own CRISPR/Cas adaptive response to evade host innate immunity.  
**First report of a CRISPR-Cas system embedded within a viral genome to counteract and bacterial host immunity defense systems**

25. CRISPR-Cas Systems RNA-mediated Adaptive Immunity in Bacteria and Archaea.  
**First book on CRISPR-Cas systems with all the knowledge accumulated up to 2013**

26. An updated evolutionary classification of CRISPR-Cas systems.  
**Updated classification of the CRISPR-Cas systems**

27. The Heroes of CRISPR  
**Perspective describing the pioneer work of many microbiologists who discovered the CRISPR-Cas systems more than 20 years ago**

28. MIMIVIRE is a defence system in mimivirus that confers resistance to virophage.  
**First report of CRISPR-Cas-like systems found inside mimiviral genomes which they use as a defense system against virofagues**

29. On the Origin of CRISPR-Cas Technology: From Prokaryotes to Mammals.  
**First review on the origin of CRISPR-Cas systems jointly prepared by a microbiologist and a mouse geneticist**

30. Diverse evolutionary roots and mechanistic variations of the CRISPR-Cas systems.  
**Great review on what we know today about the different classes of CRISPR-Cas systems in prokaryotes and their mechanisms of action**

31. A history of genome editing in mammals.  
**Review on the brief but intense history of genome editing in mammals**

```{r, fig.align='center'}
knitr::include_graphics("figures/scheme-of-CRISPR-Cas-system.PNG")
```


# Basic papers describing the use of CRISPR-Cas9 technology  
1. Multiplex genome engineering using CRISPR-Cas9 systems[@Cong819].
2. RNA-guided human genome engineering via Cas9[@Mali823].  
3. One-Step generation of mice carrying mutations in multiple genes by CRISPR-Cas mediated genome engineering[@Wang2013a].  
4. One-Step generation of mice carrying reporter and conditional alleles by CRISPR-Cas mediated genome engineering[@Yang2013].  
5. Biotechnology:Rewriting a genome[@Charpentier2013].
6. Double nicking by RNA-guided CRISPR Cas9 for enhanced genome editing specificity[@Ran2013].  
7. Genome engineering using the CRIPSR-Cas9 system[@Ran2013b].
8. RNA-guided genome editing of mammalian cells[@Storici2014].  
9. Efficient genome modification by CRISPR-Cas9 nickase with minimal off-target effects[@].  
10. Development and Applications of CRISPR-Cas9 for Genome Engineering.  
11. Generating genetically modified mice using CRISPR/Cas-mediated genome engineering.  
12. The new CRISPR-Cas system: RNA-guided genome engineering to efficiently produce any desired genetic alteration in animals.  
13. CRISPR-Cas9: Engineering a revolution in gene editing.  
14. Mouse Genome Editing Using the CRISPR/Cas System.  
15. Genome editing. The new frontier of genome engineering with CRISPR-Cas9.  
16. Redefining mouse transgenesis with CRISPR/Cas9 genome editing technology.  


# Application of CRISPR-Cas9 technology  
1. Rapid generation of mouse models with defined point mutations by the CRISPR/Cas9 system.  
2. Genome editing. The mutagenic chain reaction: a method for converting heterozygous to homozygous mutations.  
3. Functional validation of mouse tyrosinase non-coding regulatory DNA elements by CRISPR-Cas9-mediated mutagenesis.  
4. Epigenome editing by a CRISPR-Cas9-based acetyltransferase activates genes from promoters and enhancers.  
5. Unraveling CRISPR-Cas9 genome engineering parameters via a library-on-library approach.  
6. Engineered CRISPR-Cas9 nucleases with altered PAM specificities.  
7. CRISPR/Cas9-based generation of knockdown mice by intronic insertion of artificial microRNA using longer single-stranded DNA.  
8. CRISPR Inversion of CTCF Sites Alters Genome Topology and Enhancer/Promoter Function.  
9. Efficient Generation of Myostatin Knock-Out Sheep Using CRISPR/Cas9 Technology and Microinjection into Zygotes.  
10. Modeling Human Severe Combined Immunodeficiency and Correction by CRISPR/Cas9-Enhanced Gene Targeting.  
11. Genome-wide inactivation of porcine endogenous retroviruses (PERVs).  
12. Multiplexed labeling of genomic loci with dCas9 and engineered sgRNAs using CRISPRainbow.  
13. Efficient introduction of specific homozygous and heterozygous mutations using CRISPR/Cas9.  
14. CRISPR-directed mitotic recombination enables genetic mapping without crosses.  
15.  Whole organism lineage tracing by combinatorial and cumulative genome editing.  
16. Comparison of Cas9 activators in multiple species.  
17. Applications of CRISPR technologies in research and beyond.   
18. CRISPR-Based Technologies for the Manipulation of Eukaryotic Genomes.  
19. Efficient and rapid generation of large genomic variants in rats and mice using CRISMERE.  
20. Nucleic acid detection with CRISPR-Cas13a/C2c2.  
21. Targeted Deletion of an Entire Chromosome Using CRISPR/Cas9.  
22. Generation and comparison of CRISPR-Cas9 and Cre-mediated genetically engineered mouse models of sarcoma.  
23. CRISPR-Cas encoding of a digital movie into the genomes of a population of living bacteria.  
24. Genome-wide tracking of dCas9-methyltransferase footprints.  
25. Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6.  
26. Systematic discovery of antiphage defense systems in the microbial pangenome.  


# NHEJ inhibitors and other strategies for boosting the HDR pathway  
There are the first published publications reporting the use of a variety of NHEJ inhibitors in order to boost the HDR pathway when using a CRISPR-Cas9 approach, in order to improve the efficiency and success of knock-in/genome edition/substitution features versus the standard, default and far more effective NHEJ repair system, usually leading to the accumulation of indels.  

1. Small molecules enhance CRISPR genome editing in pluripotent stem cells.  
2. Increasing the efficiency of precise genome editing with CRISPR-Cas9 by inhibition of nonhomologous end joining.  
3. Increasing the efficiency of homology-directed repair for CRISPR-Cas9-induced precise gene editing in mammalian cells.  
4. RS-1 enhances CRISPR/Cas9- and TALEN-mediated knock-in efficiency.  
5. Enhancing homology-directed genome editing by catalytically active and inactive CRISPR-Cas9 using asymmetric donor DNA.  
6. Increasing the Efficiency of CRISPR/Cas9-mediated Precise Genome Editing of HSV-1 Virus in Human Cells.  
7. Marker-free coselection for CRISPR-driven genome editing in human cells.  


# CRISPR approaches in genome-wide screenings  
1. Genetic screens in human cells using the CRISPR-Cas9 system.  
2. Genome-scale CRISPR-Cas9 knockout screening in human cells.  
3. Genome-wide recessive genetic screening in mammalian cells with a lentiviral CRISPR-guide RNA library.  
4. Improved vectors and genome-wide libraries for CRISPR screening.  
5. Genome-wide CRISPR screen in a mouse model of tumor growth and metastasis.  
6. High-throughput gene targeting and phenotyping in zebrafish using CRISPR/Cas9.  
7. A Genome-wide CRISPR Screen Identifies CDC25A as a Determinant of Sensitivity to ATR Inhibitors.  
8. Directed evolution using dCas9-targeted somatic hypermutation in mammalian cells.  
9. A CRISPR Resource for Individual, Combinatorial, or Multiplexed Gene Knockout.  
10. Genome-scale activation screen identifies a lncRNA locus regulating a gene neighbourhood.  

```{r, fig.align='center', fig.cap='Scheme illustrating the most advanced CRISPR-Cas9 reagent: the RiboNucleoProtein (RNP)'}
knitr::include_graphics("figures/Scheme-illustrating-the-most-advanced-CRISPR-Cas9-reagent-the-RiboNucleoProtein(RNP).PNG")
```


# Plasmids
- hCas9 (CMV-Cas9): [Addgene #41815](http://www.addgene.org/41815/)  
- hCas9_D10A (CMV-nickase): [Addgene #41816](http://www.addgene.org/41816/)
- MLM3636 (empty sgRNA expression vector): [Addgene #43860](http://www.addgene.org/43860/)

The following two plasmids do contain the T7 promoter in their backbones. Hence, they can be used for direct in vitro RNA transcription.  

- JDS246 (CAG-T7-Cas9-NLS, Keith Joung's Lab): [Addgene #43861](http://www.addgene.org/43861/)
- pDR274 (T7-Bsal-Bsal-sgRNA scaffold, Keith Joung's Lab): [Addgene #42250](http://www.addgene.org/42250/)

# sgRNA design tools  
- http://crispr.mit.edu/
- https://crispr.med.harvard.edu/sgRNAScorer/
- https://chopchop.rc.fas.harvard.edu/index.php
- http://www.e-crisp.org/E-CRISP/
- http://www.sanger.ac.uk/htgt/wge/
- http://www.crisprscan.org/
- http://www.repeatmasker.org/cgi-bin/WEBRepeatMasker

# Protocols for CRISPR related technologies  
- http://www.genome-engineering.org/crispr/
- http://www.crispr-cas.org/
- https://crispr.med.harvard.edu/main/
- http://www.addgene.org/CRISPR/reference/#protocols
- http://www.addgene.org/CRISPR/guide/
- http://www.addgene.org/CRISPR/
- Genome engineering using the CRISPR-Cas9 system
- Generating genetically modified mice using CRISPR/Cas-mediated genome engineering
- Mouse Genome Editing Using the CRISPR/Cas System
- Generating Mouse Models Using CRISPR-Cas9-Mediated Genome Editing
- CRISPR/Cas9-mediated gene manipulation to create single-amino-acid-substituted and floxed mice with a cloning-free method


# Resources  
- http://wwwuser.cnb.csic.es/~montoliu/CRISPR/
- https://benchling.com/pub/liu-base-editor

# REFERENCE
